### DISCUSSION TITLE
can mitsuba be used on mobile devices?

### DISCUSSION THREAD

**Discussion by anewusername77** [mitsuba-renderer/mitsuba3#290] [Q&A]
without intel CPU and nvidia GPU.

best regards.

**Comment by Speierers** [comment]
It will be hard to properly answer this question with so little information and context unfortunately. 

If you can install LLVM on your mobile device you might be able to use the llvm variants, otherwise you should be able to compile the scalar variant. Never tried it myself and this will likely be too slow for any practical application as it won't use the mobile GPU chip.

**Comment by anewusername77** [comment]
thanks, it helps a lot!
